lisa/sut_orchestrator/azure/transformers.py

```diff
@@ -178,22 +182,38 @@ def _prepare_virtual_machine(self, node: RemoteNode) -> None:
         startstop = node.features[StartStop]
         startstop.stop()
 
-    def _export_vhd(self, platform: AzurePlatform, virtual_machine: Any) -> str:
+    def _export_vhd(
+        self, platform: AzurePlatform, virtual_machine: Any
+    ) -> Tuple[str, str]:
         runbook: VhdTransformerSchema = self.runbook
         compute_client = get_compute_client(platform)
 
         # generate sas url from os disk, so it can be copied.
         self._log.debug("generating sas url...")
         location = virtual_machine.location
         os_disk_name = virtual_machine.storage_profile.os_disk.name
+
+        is_cvm = (
+            virtual_machine.storage_profile.os_disk.managed_disk.security_profile
```

Comment:
Is it accurate? Why security file is not None equals CVM?

Reply:
Maybe it would be more clear if the variable name was `has_vmgs`. That is what I am actually checking here. But yes, it is accurate that only a CVM will have a security profile on the OS Disk.